02-household-queries/qt_summaries-gpt-3.5-turbo-instruct-k_4.json

```diff
@@ -0,0 +1,851 @@
+[
+    {
+        "id": 1,
+        "question": "The client has her 19 year old grandchild living with her. She says she does her own thing and they don't really share food. Does the grandchild need to be included?",
+        "guru_cards": [
+            "Who are mandatory HH members for food stamps?"
+        ],
```

Comment:
`guru_cards` is the set of expected cards to be cited according to https://navasage.atlassian.net/wiki/spaces/NL/pages/1053229170/E2+Tool+Inputs+and+Required+Data.